### DISCUSSION TITLE
Can't re-use .quant files

### DISCUSSION THREAD

**Discussion by mdubuke** [vdemichev/DiaNN#878] [Q&A]
Hi all!

I'm attempting to re-use quant files after the original files have been removed from my server, and I'm getting this error: 

_[0:52] File #109/109
[0:52] Loading run /data/EXP230113/EXP230113_PRTC_Slot1-95_1_1008.d
stat lib: No such file or directory
addr2line: Reading source function failed for module diann
ERROR: cannot open the raw data folder /data/EXP230113/EXP230113_PRTC_Slot1-95_1_1008.d
ERROR: either the dia-PASEF file is damaged or a .dia file produced by an older DIA-NN version has been loaded: performance might be suboptimal, regenerate the .dia file using this DIA-NN version
WARNING: incorrectly recorded isolation window margins
[0:52] 0 library precursors are potentially detectable
[0:52] Processing...
[0:52] Using MS1 mass accuracy: 20 ppm
[0:52] Using mass accuracy: 20 ppm
[0:52] Removing low confidence identifications
[0:52] Removing interfering precursors
[0:52] Too few confident identifications, neural networks will not be used
[0:52] Number of IDs at 0.01 FDR: 0
[0:52] Calculating protein q-values
[0:52] Number of protein isoforms identified at 1% FDR: 0 (precursor-level), 0 (protein-level) (inference performed using proteotypic peptides only)
[0:52] Quantification_

This is command line call (just showing the last file to cleanliness, but all the rest follow the same pattern):

_diann --f "/data/EXP230113/EXP230113_PRTC_Slot1-95_1_1008.d" --lib "/data/EXP230113/library.tsv" --verbose 1 --out "/data/EXP230113/DIANN-Norm/report.tsv" --qvalue 0.01 --temp "/data/EXP230113/DIANN/" --reanalyse --relaxed-prot-inf --mod [CustomMod], [Custom Mass] --use-quant_

Any help would be greatly appreciated!!

--Michelle

**Comment by ramosemi** [comment]
I am getting the same error. Have never had trouble running DIA-NN before...
incorrectly recorded isolation window margins.

diann.exe --f "C:\Users\Mremi\Downloads\231206_ATG7KO_rep1_NC_Slot1-1_1_915.d
" --lib "" --threads 6 --verbose 1 --out "C:\DIA-NN\1.8.1\H3.tsv" --qvalue 0.01 --matrices  --out-lib "C:\DIA-NN\1.8.1\report-lib.tsv" --gen-spec-lib --predictor --fasta "C:\Users\Mremi\Documents\Raw data\1093\SWISSPROT-HUMAN-20350ENTRIES-with CALR mut.fasta" --fasta-search --min-fr-mz 200 --max-fr-mz 1800 --met-excision --cut K*,R* --missed-cleavages 0 --min-pep-len 7 --max-pep-len 30 --min-pr-mz 300 --max-pr-mz 1800 --min-pr-charge 1 --max-pr-charge 4 --unimod4 --mass-acc 10 --mass-acc-ms1 20 --double-search --relaxed-prot-inf --smart-profiling --peak-center 
DIA-NN 1.8.1 (Data-Independent Acquisition by Neural Networks)
Compiled on Apr 14 2022 15:31:19
Current date and time: Sun Dec 10 10:59:34 2023
CPU: GenuineIntel Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
SIMD instructions: AVX AVX2 FMA SSE4.1 SSE4.2 
Logical CPU cores: 12
Thread number set to 6
Output will be filtered at 0.01 FDR
Precursor/protein x samples expression level matrices will be saved along with the main report
A spectral library will be generated
Deep learning will be used to generate a new in silico spectral library from peptides provided
Library-free search enabled
Min fragment m/z set to 200
Max fragment m/z set to 1800
N-terminal methionine excision enabled
In silico digest will involve cuts at K*,R*
Maximum number of missed cleavages set to 0
Min peptide length set to 7
Max peptide length set to 30
Min precursor m/z set to 300
Max precursor m/z set to 1800
Min precursor charge set to 1
Max precursor charge set to 4
Cysteine carbamidomethylation enabled as a fixed modification
Neural networks will be used for peak selection
Highly heuristic protein grouping will be used, to reduce the number of protein groups obtained; this mode is recommended for benchmarking protein ID numbers; use with caution for anything else
When generating a spectral library, in silico predicted spectra will be retained if deemed more reliable than experimental ones
Fixed-width center of each elution peak will be used for quantification
Mass accuracy will be fixed to 1e-05 (MS2) and 2e-05 (MS1)
Exclusion of fragments shared between heavy and light peptides from quantification is not supported in FASTA digest mode - disabled; to enable, generate an in silico predicted spectral library and analyse with this library

1 files will be processed
[0:00] Loading FASTA C:\Users\Mremi\Documents\Raw data\1093\SWISSPROT-HUMAN-20350ENTRIES-with CALR mut.fasta
WARNING: 3 sequences skipped due to duplicate protein ids; use --duplicate-proteins to disable skipping duplicates
[0:01] Processing FASTA
[0:04] Assembling elution groups
[0:06] 1604063 precursors generated
[0:06] Gene names missing for some isoforms
[0:06] Library contains 20340 proteins, and 20112 genes
[0:06] Encoding peptides for spectra and RTs prediction
[0:08] Predicting spectra and IMs
[9:39] Predicting RTs
[10:34] Decoding predicted spectra and IMs
[10:39] Decoding RTs
[10:40] Saving the library to C:\DIA-NN\1.8.1\report-lib.predicted.speclib
[10:43] Initialising library

[10:44] File #1/1
[10:44] Loading run C:\Users\Mremi\Downloads\231206_ATG7KO_rep1_NC_Slot1-1_1_915.d
For most diaPASEF datasets it is better to manually fix both the MS1 and MS2 mass accuracies to values in the range 10-15 ppm.
ERROR: cannot open the raw data folder C:/Users/Mremi/Downloads/231206_ATG7KO_rep1_NC_Slot1-1_1_915.d
ERROR: cannot open the raw data folder C:/Users/Mremi/Downloads/231206_ATG7KO_rep1_NC_Slot1-1_1_915.d
ERROR: cannot open the raw data folder C:/Users/Mremi/Downloads/231206_ATG7KO_rep1_NC_Slot1-1_1_915.d
ERROR: cannot open the raw data folder C:/Users/Mremi/Downloads/231206_ATG7KO_rep1_NC_Slot1-1_1_915.d
ERROR: cannot open the raw data folder C:/Users/Mremi/Downloads/231206_ATG7KO_rep1_NC_Slot1-1_1_915.d
ERROR: either the dia-PASEF file is damaged or a .dia file produced by an older DIA-NN version has been loaded: performance might be suboptimal, regenerate the .dia file using this DIA-NN version
WARNING: incorrectly recorded isolation window margins
[10:46] 0 library precursors are potentially detectable
[10:46] Processing...
[10:46] Removing low confidence identifications
[10:46] Removing interfering precursors
[10:46] Too few confident identifications, neural networks will not be used
[10:46] Number of IDs at 0.01 FDR: 0
[10:46] Calculating protein q-values
[10:46] Number of genes identified at 1% FDR: 0 (precursor-level), 0 (protein-level) (inference performed using proteotypic peptides only)
[10:46] Quantification
[10:46] Quantification information saved to C:\Users\Mremi\Downloads\231206_ATG7KO_rep1_NC_Slot1-1_1_915.d.quant.

**Comment by vdemichev** [comment]
Hi Michelle,

This means .quant files with proper names are not found in the folder DIA-NN in which is looking for them. The naming system is a bit complex (apologies for that, will probably rework), so to figure out how it works the easiest is to run with proper raw files, see where .quant files are saved and how they are named, and then you have an idea how you should reference the (nonexistent) .raw files to reuse the .quant files.

Best,
Vadim

**Comment by vdemichev** [comment]
Hi Emilio,

In your case it seems it just cannot open the Bruker .d folder for some reason. If you copy the Bruker folder to C:/Temp, does it work? Please also check it does contain both .tdf and .tdf_bin files inside.

Best,
Vadim

**Comment by ramosemi** [comment]
Thank you, Vadim!
What a pleasure to meet you. I love DIA-NN and it is such an incredible tool.

We (Bruker field application scientist) figured out that our timsTOF quadropole is out of calibration and was causing an error in DIA-NN (too few precursors, cannot calibrate).

Everything is working well again with mass accuracy/MS1 both set to 100 so these samples aren’t lost and we get the quadrupole calibrated.

I didn't know I could choose such a high mass accuracy but te data look great (very similar to TMT on the same set of samples).

Thank you again for helping,
Emilio

________________________________

**Reply by vdemichev** [reply]
Hi Emilio,

Can you please also try --mass-acc-cal 100, while keeping regular mass accuracies fixed to 15ppm?

Best,
Vadim

**Reply by vdemichev** [reply]
Might improve results a lot

**Comment by mdubuke** [comment]
Hi Vadim,

Thanks so much for this; I see now how the files are named to reference
where the raw file originally was. I will re-work my filename calls to
adjust!

--Michelle
